examples/connectivity/plot_dict_learning_resting_state.py

```diff
@@ -0,0 +1,71 @@
+"""
+Group analysis of resting-state fMRI with dictionary learning: DictLearning
+=====================================================
+
+An example applying dictionary learning to resting-state data. This example applies it
+to 10 subjects of the ADHD200 datasets.
+
+Dictionary learning is a sparsity based decomposition method for extracting spatial maps.
+
+    * Gael Varoquaux et al.
+    Multi-subject dictionary learning to segment an atlas of brain spontaneous activity
+    Information Processing in Medical Imaging, 2011, pp. 562-573, Lecture Notes in Computer Science
+
+Pre-prints for paper is available on hal
+(http://hal.archives-ouvertes.fr)
+"""
+
+### Load ADHD rest dataset ####################################################
+from nilearn import datasets
+# For linear assignment (should be moved in non user space...)
+
+adhd_dataset = datasets.fetch_adhd(n_subjects=20, data_dir='/media/data/neuro')
```

Comment:
You cannot change the data_dir in an example. This crashes on people's computer.

My advice: just make a symlink on your computer from ~/nilearn_data to whereever is convenient.